test/integration/clusterio.js

```diff
@@ -1,14 +1,39 @@
+/* eslint-disable no-console */
```

Comment:
With a handful of exceptions there should be no use of console in the project, this is one of the files it shouldn't be used in.

Reply:
i am using the spawn method from index.js which uses it